### DISCUSSION TITLE
版本5为授权的小程序帐号上传小程序代码如何使用的，一直没有找到使用方法？？

### DISCUSSION THREAD

**Discussion by zjj10528272** [w7corp/easywechat#2734] [General]
版本5为授权的小程序帐号上传小程序代码如何使用的，一直没有找到使用方法？？

**Comment by overtrue** [comment]
没有这个封装

**Reply by zjj10528272** [reply]
但是为何代码里面是有这个方法的
![image](https://github.com/w7corp/easywechat/assets/74055370/48dcecee-c9c2-42ea-9f01-696045714f3a)

**Reply by zjj10528272** [reply]
已找到使用办法
![image](https://github.com/w7corp/easywechat/assets/74055370/569e9253-3f12-4724-87ad-a78fddcc85b0)
